src/data/endpoints/account.yaml

```diff
@@ -36,23 +34,39 @@ endpoints:
                   "email": "jsmith@mycompany.com",
                   "first_name": "John",
                   "last_name": "Smith",
-                  "network_helper": true,
                   "phone": "555-555-1212",
                   "state": "PA",
                   "zip": 19102,
+                  }
+                }' \
+                https://$api_root/$version/account
+  /account/settings:
+    group: Settings
+    type: resource
+    description: >
+      Manage default settings for your account.
+    methods:
+      GET:
+        response: account_settings
+        description: >
+          Returns your current defaults.
+        examples:
+          curl: |
+            curl -h "Authorization: Bearer $TOKEN" \
+                https://$api_root/$version/account/settings
+      PUT:
+        description: >
+          Edits your account's defaults.
+        examples:
+          curl: |
+            curl -h "Content-Type: application/json" \
+                -h "Authorization: Bearer $TOKEn" \
+                -X PUT -d '{
+                  "network_helper": true,
                   "longview_subscription": "longview-10"
```

Comment:
So there is nothing else to edit in account/settings now?

If there is, please add them. If not, then, uh... cool!

Reply:
with this proposal, there is not